Demo/Demo/Features/CardDemoViewController.swift

```diff
@@ -0,0 +1,106 @@
+        static let textFieldHeight: CGFloat = 40
+    }
+
+    let mainStackView: UIStackView = {
+        let view = UIStackView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.axis = .vertical
+        view.alignment = .center
+        view.spacing = Constants.stackViewSpacing
+        return view
+    }()
+
+    let secondaryStackView: UIStackView = {
+        let view = UIStackView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.axis = .horizontal
+        view.alignment = .center
+        view.distribution = .fillEqually
+        view.spacing = Constants.stackViewSpacing
+        return view
+    }()
+
+    lazy var cardNumberTextField: CustomTextField = {
+        let textField = CustomTextField(placeholderText: "Card Number")
+        textField.keyboardType = .numberPad
+        return textField
+    }()
+
+    lazy var expirationTextField: CustomTextField = {
+        let textField = CustomTextField(placeholderText: "Expiration")
+        textField.keyboardType = .numberPad
+        return textField
+    }()
+
+    lazy var cvvTextField: CustomTextField = {
+        let textField = CustomTextField(placeholderText: "CVV")
+        textField.keyboardType = .numberPad
+        return textField
+    }()
+
+    lazy var payButton: UIButton = {
+        let payButton = CustomButton(title: "\(DemoSettings.intent.rawValue.capitalized) Order")
+        payButton.addTarget(self, action: #selector(didTapPayButton), for: .touchUpInside)
+        payButton.layer.cornerRadius = 8
+        payButton.backgroundColor = .systemBlue
+        payButton.tintColor = .white
+        return payButton
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = .systemBackground
+
+        setupUI()
+        setupConstraints()
+    }
+
+    func setupUI() {
+        view.addSubview(mainStackView)
+
+        mainStackView.addArrangedSubview(cardNumberTextField)
+        mainStackView.addArrangedSubview(secondaryStackView)
+        mainStackView.addArrangedSubview(payButton)
+
+        secondaryStackView.addArrangedSubview(expirationTextField)
+        secondaryStackView.addArrangedSubview(cvvTextField)
+    }
+
+    func setupConstraints() {
+        NSLayoutConstraint.activate([
+            mainStackView.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+            mainStackView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+
+            secondaryStackView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Constants.layoutSpacing),
+            secondaryStackView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -Constants.layoutSpacing),
+
+            cardNumberTextField.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Constants.layoutSpacing),
+            cardNumberTextField.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -Constants.layoutSpacing),
```

Comment:
since `cardNumberTextField` is an arranged subview inside `mainStackView`, I think it'll be sized properly by the stackview so we may not need `leading/trailingAnchor` for this.

Reply:
instead we can have constraints for `mainStackView` https://github.com/paypal/iOS-SDK/pull/14#discussion_r722453787 and it will be applied to all arranged subviews :smile: